app/web/features/dashboard/CommunityBrowser.tsx

```diff
@@ -10,36 +10,51 @@ import {
 import { useTranslation } from "i18n";
 import { DASHBOARD } from "i18n/namespaces";
 import { Community } from "proto/communities_pb";
-import { useEffect, useRef, useState } from "react";
+import { useState } from "react";
 import { routeToCommunity } from "routes";
-import makeStyles from "utils/makeStyles";
 
-const useStyles = makeStyles((theme) => ({
-  root: {
-    display: "flex",
-    flexDirection: "row",
-    alignItems: "flex-start",
-    "& > * + *": {
-      marginInlineStart: theme.spacing(2),
-    },
+const OuterWrapper = styled("div")(({ theme }) => ({
+  display: "flex",
+  flexDirection: "row",
+  alignItems: "flex-start",
+  width: "100%",
+  "& > * + *": {
+    marginInlineStart: theme.spacing(2),
```

Comment:
Thank you so much for converting this to the new format! Saves me so much time later!